jekyll/_cci2/reusing-config.md

```diff
@@ -625,6 +611,93 @@ workflows:
 ```
 --->
 
+### Using Parameters in Executors
+{:.no_toc}
+
+To use parameters in executors, define the parameters under the given executor. When you invoke the executor, pass the keys of the parameters as a map of keys under the `executor:` declaration, each of which has the value of the parameter to pass in.
+
+Parameters in executors can be of the type `string`, `enum`, or `boolean`. Default values can be provided with the optional `default` key.
+
+#### Example Build Configuration Using a Parameterized Executor
+{:.no_toc}
+
+```yaml
+version: 2.1
+
+executors:
+  python:
+    parameters:
+      tag:
+        type: string
+        default: latest
+      myspecialvar:
+        type: string
+    docker:
+      - image: circleci/python:<< parameters.tag >>
+    environment:
+      MYPRECIOUS: << parameters.myspecialvar >>
+
+jobs:
+  build:
+    executor:
+      name: python
+      tag: "2.7"
+      myspecialvar: "myspecialvalue"  
+```
+
+The above would resolve to the following: 
+
+```yaml
+version: 2.1
+jobs:
+  build:
+    steps: []
+    docker:
+      - image: circleci/python:2.7
+    environment:
+      MYPRECIOUS: "myspecialvalue"
+```
+
+
+
+### The Scope of Parameters
+{:.no_toc}
+
+Parameters are in-scope only within the job or command that defined them. If you want a job or command to pass its parameters to a command it invokes, they must be passed explicitly. Command, job, executor, and parameter names can only contain lowercase letters a-z, digits, and _ and -, and must start with a letter.
```

Comment:
Re: rules about names, true now, but I thought we were in the process of changing this?